awxkit/awxkit/api/pages/api.py

```diff
@@ -13,7 +55,249 @@ class Api(base.Base):
 
 class ApiV2(base.Base):
 
-    pass
+    # Export methods
+
+    def _export(self, _page, post_fields):
+        # Drop any (credential_type) assets that are being managed by the Tower instance.
+        if _page.json.get('managed_by_tower'):
```

Comment:
I've wanted for some time now to try to refactor all of this so that Page and PageList get export methods and this all isn't just piled up here on the ApiV2 class.  Each of my attempts so far has been a failure, though.  The export of a single object turns out to be annoyingly context specific.